enclave-runner/src/usercalls/mod.rs

```diff
@@ -644,8 +634,15 @@ impl EnclaveState {
     ) -> StdResult<(u64, u64), EnclaveAbort<EnclavePanic>> {
         let (tx_return_channel, mut rx_return_channel) = tokio::sync::mpsc::unbounded_channel();
         let enclave_clone = enclave.clone();
+        let mut rt = tokio::runtime::Builder::new()
+            .basic_scheduler()
```

Comment:
As I understand basic_scheduler means everything runs on a single thread.
It's what_ we are aiming for.
But do we want to see if a (multi) threaded_scheduler improves performance?

Reply:
We can experiment with that, but I don't expect that would help. This might actually explain why we saw lower performance for tokio 0.2 implementation over 0.1 before, as we were using multi-threaded scheduler during that test for 0.2.